src/main/fc/core.c

```diff
@@ -279,7 +279,7 @@ void updateArmingStatus(void)
 
 #ifdef USE_GPS_RESCUE
         if (gpsRescueIsConfigured()) {
-            if (!gpsRescueConfig()->minSats || STATE(GPS_FIX) || ARMING_FLAG(WAS_EVER_ARMED)) {
+            if (gpsRescueConfig()->allowArmingWithoutFix || STATE(GPS_FIX)) {
```

Comment:
This should still check `ARMING_FLAG(WAS_EVER_ARMED)` - otherwise it will make it possible for the pilot to land / disarm in an inaccessible location and then not being able to get out of there again if there is no GPS reception.